modules/openapi-generator/src/main/resources/csharp-netcore/appveyor.mustache

```diff
@@ -0,0 +1,13 @@
+# auto-generated by OpenAPI Generator (https://github.com/OpenAPITools/openapi-generator)
+#
+image: Visual Studio 2019
+clone_depth: 1
+before_build:
+- dotnet restore
+build:
+  project: {{{packageName}}}.sln
+  verbosity: minimal
```

Comment:
```suggestion
build_script:
- dotnet build -c Release
```

Since the appveyor build command behaves differently on Windows and Linux, it is better to use the dotnet build command.